source/libs/page-detect.ts

```diff
@@ -103,7 +106,7 @@ export const _isGlobalSearchResults = [
 	'https://github.com/search?q=refined-github&ref=opensearch'
 ];
 
-export const isIssue = (): boolean => /^issues\/\d+/.test(getRepoPath()!);
+export const isIssue = (): boolean => /^issues\/\d+/.test(getRepoPath()!) && document.title !== 'GitHub · Where software is built';
```

Comment:
Add a comment explaining what the second part means: why is it there? When does it happen?
```suggestion
export const isIssue = (): boolean => /^issues\/\d+/.test(getRepoPath()!) && document.title !== 'GitHub · Where software is built'; // The title check excludes deleted issues
```